miasm2/core/asmblock.py

```diff
@@ -67,8 +67,17 @@ def set_label(self, loc_key):
 
     label = property(get_label, set_label)
 
+    def to_string(self, symbol_pool=None):
+        if symbol_pool is None:
+            return "%s:%s" % (str(self.c_t), str(self.loc_key))
```

Comment:
`str`s are useless, due to string format

Reply:
done